### ISSUE TITLE
Provide a browser bundle suitable for direct browser consumption

### ISSUE THREAD

**Issue by schrepfler** [apollographql/subscriptions-transport-ws#47]
Noticed the client uses `require` which makes sense if one uses it on node or via browserify/build tools. What if we also expose a browser bundle for the client which already ships everything packaged so that it's possible to reference the library directly via https://unpkg.com?

**Comment by stubailo** [comment]
@schrepfler if you could send a PR with a script that uses Webpack or Browserify to generate a bundle so that we can use `unpkg`, that would be great!

**Comment by AlexLeung** [comment]
I use `unpkg` all the time. This would be really helpful!

**Comment by schrepfler** [comment]
I think this is the basis https://github.com/apollostack/subscriptions-transport-ws/pull/48 but I need some validation as to 1) is the generated bundle consumable from a </script> tag  and will the configuration publish the bundle onto npm or do I need to add more stuff to the package.json (I'm not really a JS dev so all this module mumbo jumbo seems a bit like black magic.)

**Comment by schrepfler** [comment]
Although I think we merged the support for this, I can see https://unpkg.com/subscriptions-transport-ws@0.5.3 contains require statements (which I don't have locally.)
ex. locally the client.js transpiled bundle starts with
```
var SubscriptionsTransportWs =
/******/ (function(modules) { // webpackBootstrap
/******/        // The module cache
/******/        var installedModules = {};

/******/        // The require function
/******/        function __webpack_require__(moduleId) {

/******/                // Check if module is in cache
/******/                if(installedModules[moduleId])
/******/                        return installedModules[moduleId].exports;

/******/                // Create a new module (and put it into the cache)
/******/                var module = installedModules[moduleId] = {
/******/                        i: moduleId,
/******/                        l: false,
/******/                        exports: {}
/******/                };
```

**Comment by schrepfler** [comment]
Actually, there seems to have been a regression, 0.5.2 was generating a different output: https://unpkg.com/subscriptions-transport-ws@0.5.2

**Comment by schrepfler** [comment]
Went through the changes for 0.5.3 and noticed the change in use of unpkg and that I need to explicitly use https://unpkg.com/subscriptions-transport-ws@0.5.3/browser/client.js 
I think this beats the purpose of the "browser" node which should specify the default browser main entry point, if that was kept as-is https://unpkg.com/subscriptions-transport-ws@0.5.3 would be enough to specify?

**Comment by Urigo** [comment]
Fixed in `0.5.1`.
